src/constants.ts

```diff
@@ -52,19 +52,6 @@ export const providers: any = {
     fuji: new ethers.providers.JsonRpcProvider(urls.provider.fuji)
 }
 
-export const addresses: any = {
```

Comment:
I think this might also be a good PR to remove the bytecode hashes and functions that use the bytecode hash to compute the option chain address since we no longer need that functionality.

Reply:
done